rosa_planning/rosa-limits-scalability.adoc

```diff
@@ -22,4 +22,4 @@ include::modules/sd-planning-considerations.adoc[leveloffset=+1]
 [id="additional-resources_rosa-limits-scalability"]
 == Additional resources
 
-* xref:../logging/sd-accessing-the-service-logs.adoc#sd-accessing-the-service-logs[Accessing the service logs for ROSA clusters]
+* xref:../observability/logging/sd-accessing-the-service-logs.adoc#sd-accessing-the-service-logs[Accessing the service logs for ROSA clusters]
```

Comment:
Actual section is titled "Accessing the service logs for Red Hat OpenShift Service on AWS clusters"

Reply:
Please see my https://github.com/openshift/openshift-docs/pull/74819#discussion_r1577387726.